var/spack/repos/builtin/packages/nccl-fastsocket/package.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import tempfile
+
+from spack.package import *
+
+
+class NcclFastsocket(Package):
+    """NCCL Fast Socket GCP Net plugin for NCCL"""
+
+    homepage = "https://github.com/google/nccl-fastsocket"
+    git = "https://github.com/google/nccl-fastsocket.git"
+
+    version("master", preferred=True)
+
+    depends_on("bazel", type="build")
+    depends_on("nccl", type=["build", "run"])
+
+    maintainers = ["danielahlin"]
```

Comment:
The maintainers syntax has changed (see https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers).

```suggestion
    maintainers("danielahlin")
```